tests/functional/firefox/test_notes.py

```diff
@@ -0,0 +1,113 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.accounts import FirefoxNotesPage
+
+
+@pytest.mark.nondestructive
+def test_firefox_desktop_release_page(base_url, selenium):
+    #firefox/65.0/releasenotes/
+    page = FirefoxNotesPage(selenium, base_url).open()
+    assert page.version is '65.0'
+    assert page.channel is 'Firefox Release'
+    assert page.gtm_id is '/firefox/releasenotes/'
+    assert page.download_button_text is 'Download Firefox'
+    assert page.download_button_link.contains('firefox-stub')
+    assert page.logo is '/media/img/logos/firefox/logo-quantum-wordmark-large.png'
```

Comment:
This may pass locally but will fail in production when the filepath contains a hash. It's also a bit overly specific for a functional test. For example if we move or rename the file then the test could fail even though the right thing is still shown on the page.

I'd suggest asserting that the image is displayed on the page, rather than the filepath used in the HTML.

Reply:
I want to test that the correct logos are appearing based on the product, channel, and version. How would you suggest I check that?

Reply:
Use a CSS selector to target the image, and then locate that element and assert it is displayed on the page.